docs/reference/intro.asciidoc

```diff
@@ -88,94 +122,6 @@ The analysis chain that is applied to a full-text field during indexing is also
 used at search time. When you query a full-text field, the query text undergoes
 the same analysis before the terms are looked up in the index.
 
-[[search-analyze]]
-=== Information out: search and analyze
```

Comment:
afraid to lose this search overview here. I don't think it's good but I don't think we're adequately pointing to a replacement ATM

Reply:
yeah we have no backlinks in here though so I was happy to cut it out and do a full replacement later but I can leave it in